kratos/solving_strategies/schemes/residual_based_bdf_scheme.h

```diff
@@ -121,7 +122,8 @@ class ResidualBasedBDFScheme
      */
     explicit ResidualBasedBDFScheme(const std::size_t Order = 2)
         :ImplicitBaseType(),
-         mOrder(Order)
+         mOrder(Order),
+         mpBDFUtility(Kratos::make_shared<TimeDiscretization::BDF>(Order))
```

Comment:
Ok if it has to be a pointer then pls use unique

```suggestion
         mpBDFUtility(Kratos::make_unique<TimeDiscretization::BDF>(Order))
```